ui/dashboards/src/components/PanelDrawer/PanelEditorForm.tsx

```diff
@@ -72,8 +73,13 @@ export function PanelEditorForm(props: PanelEditorFormProps) {
 
   return (
     // Grid maxHeight allows user to scroll inside Drawer to see all content
-    <form id={panelEditorFormId} onSubmit={handleSubmit}>
-      <Grid container spacing={2} sx={{ overflowY: 'scroll', maxHeight: '90vh' }}>
+    <Box
+      component="form"
+      id={panelEditorFormId}
+      onSubmit={handleSubmit}
+      sx={{ flex: 1, overflowY: 'scroll', padding: (theme) => theme.spacing(2) }}
```

Comment:
This adjustment: 
- enables the content of the panel to fill all the space not used by the header and scroll
- moves the padding down here (from the top level drawer), so that the scrollbar sits snugly against the right of the browser window